articles/logic-apps/authenticate-with-managed-identity.md

```diff
@@ -520,7 +520,7 @@ Before you can use your logic app's managed identity for authentication, you hav
 > suppose you have a managed identity for a logic app that needs access to update the application 
 > settings for that same logic app from a workflow. You must give that identity access to the associated logic app.
 
-For example, to access an Azure Blob storage account or an Azure key vault with your managed identity, you need to set up Azure role-based access control (Azure RBAC) and assign the appropriate role for that identity to the storage account or key vault, respectively.
+For example, to access an Azure Blob storage account or an Azure Key Vault with your managed identity, you need to set up Azure role-based access control (Azure RBAC) and assign the appropriate role for that identity to the storage account or key vault, respectively.
```

Comment:
Hi @changeworld,

"Azure Key Vault" is the service name, while a "key vault" is the resource or instance. So, if you want, change to the following:

```suggestion
For example, to authenticate with a managed identity for access to a Blob storage account or key vault in Azure, you need to set up Azure role-based access control (Azure RBAC) and assign the appropriate role for that identity to the storage account or key vault, respectively.
```